roboteam_ai/src/skills/Dribble.cpp

```diff
@@ -11,7 +11,7 @@ Dribble::Dribble(string name, bt::Blackboard::Ptr blackboard)
 
 Dribble::Progression Dribble::checkProgression() {
     if (currentProgress == ON_THE_WAY) {
-        if (! robotHasBall()) {
+        if (! World::ourBotHasBall(robot->id,Constants::MAX_BALL_BOUNCE_RANGE())) {
```

Comment:
Is it really necessary to pass this constant on. Should the function be able to get it from Constants itself?

Reply:
Well, yes, because we have 2 definitions of whether a robot has a ball or not.  (one at 0.05, with bounces allowed up until 0.15 m) This way we can also use this easily in the future for for example checking if a robot can kick a ball easily by passing a constants::MAX_KICKING_DIST or so.